ts/Dashboards/Components/Component.ts

```diff
@@ -838,6 +870,32 @@ abstract class Component {
         return this;
     }
 
+    /**
+     * Removes the component instance from the provided connector.
+     * Stops the connector polling if the last element is removed.
+     *
+     * @param connector
+     * Instance of a connector.
+     *
+     * @internal
+     */
+    public removeConnectorAssignment(
+        connector: Component.ConnectorTypes | undefined
```

Comment:
`connector?: Component.ConnectorTypes`

Its optional then and we do not need to use "undefined type"